### ISSUE TITLE
Deprecation warning in `local-testing-mode`

### ISSUE THREAD

**Issue by kaelynj** [Qiskit/documentation#1845]
### URL to the relevant documentation

https://docs.quantum.ibm.com/guides/local-testing-mode

### Select all that apply

- [ ] typo
- [x] code bug
- [ ] out-of-date content
- [ ] broken link
- [ ] other

### Describe the fix.

In the penultimate cell of the "local testing mode" docs page, there is a deprecation warning that pops up when calling `AerSimulator.from_backend()` from a hardware backend provided through `qiskit-ibm-runtime`:

```
DeprecationWarning:` The class ``qiskit.providers.models.backendconfiguration.QasmBackendConfiguration`` is deprecated as of qiskit 1.2. It will be removed in the 2.0 release. The models in ``qiskit.providers.models`` are part of the deprecated `BackendV1` workflow and no longer necessary for `BackendV2`. If a user workflow requires these representations it likely relies on deprecated functionality and should be updated to use `BackendV2`.
  aer = AerSimulator.from_backend(real_backend)
```

I assume this has to do with how aer is constructing backends, but need to investigate further what the fix is.

**Comment by beckykd** [comment]
Contacted @t-imamichi to determine whether that team was aware of the issue, and whether it was being addressed.

**Comment by t-imamichi** [comment]
It has been already fixed and released as qiskit-aer 0.15. Could you upgrade qiskit-aer?

I suggest writing versions of qiskit and qiskit-aer (and python and OS) when you report a bug.
It is hard to reproduce and fix bugs without version information.
